src/RenderWebGL.js

```diff
@@ -182,9 +182,23 @@ class RenderWebGL extends EventEmitter {
         /** @type {function} */
         this._exitRegion = null;
 
+        /** @type {object} */
+        this._backgroundDrawRegionId = {
+            enter: () => this._enterDrawBackground(),
+            exit: () => this._exitDrawBackground()
+        };
+
         /** @type {Array.<snapshotCallback>} */
         this._snapshotCallbacks = [];
 
+        /** @type {Array<number>}
+         * @readonly */
+        this._backgroundColor4f = [0, 0, 0, 0];
```

Comment:
Because this says readonly, it looks to me like background color is black by default and will always be black, so might be better to have these initialize to white and drop the call to setBackgroundColor